### PR TITLE
contact section is updated now on nav bar

### PR DESCRIPTION
**Pull request by Sushilverma002**
## Description 📝

Fixes/Closes: #294 
contact section is updated now on nav bar
## Changes 💬

- High-level overview of the changes introduced.
- Step-by-step breakdown of the implementation, if applicable.

## Screenshots (if applicable)
image

Include relevant screenshots or GIFs demonstrating the changes, especially for UI-related updates.

## Additional Notes 📒

Include any additional information or context that might be helpful for the reviewers. 
***Also if you are fixing the already made PR and the reviewer demands some change in it then, in each PR you have to mention the issue you have fixed in it.***

### CODE REVIEW

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

File: Components/LandingPage/LandingNav.tsx [hunk 1]
```diff
@@ -35,6 +35,12 @@ const LandingNav = () => {
       benefitsSection.scrollIntoView({ behavior: "smooth" });
     }
   };
+  const handlecontactusclick = () => {
+    const contactsection = document.getElementById("contactUs");
+    if (contactsection) {
+      contactsection.scrollIntoView({ behavior: "smooth" });
+    }
+  };
```
[the remark this answers is not in the document]
Comment (Sushilverma002) [reply Components/LandingPage/LandingNav.tsx]: sir everything seem to be correct but it doesn't move smoothly similar in our service section it not going smooth but still it was their.

**Comment by Sushilverma002** [comment]
updated please check @PiyushKalyanpy

**Comment by PiyushKalyanpy** [comment]
You have referred to the wrong Issue!!

**Comment by Sushilverma002** [comment]
I am unable to detect where I'm going wrong. Smoothness while scrolling is not working properly. Otherwise everything seems fit.

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by PiyushKalyanpy** [comment]
Hey there, 
We are closing this pull request as the changes we requested were not implemented. Please create a new PR once you implement the changes.

**Pull request closed without merging**